bundle.md

```diff
@@ -10,7 +10,13 @@ A standard container bundle is made of the following 3 parts:
 - One or more content directories
 - A configuration file
 
-# Directory layout
+## Use Cases
+
+This bundle specification enables users in the following cases:
+
+- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runner.
```

Comment:
On Wed, Jul 29, 2015 at 09:38:06AM -0700, Mrunal Patel wrote:
> > +- A user can create a root filesystem and configuration, with
> > low-level OS and host specific details, and launch it as a
> > container under an Open Container runner.
> 
> Maybe "implementation" instead or "runner" ?

The current spec seems to use both ‘runtime’, ‘runc’, and
‘implementation’.  I don't have a strong preference for any of these
choices, but it would be nice to pick a single name to standardize on.
Since ‘runtime’ is currently the most popular (by grep hits), can we
use that?